src/lsp/gdscript.capabilities.ts

```diff
@@ -34,3 +35,9 @@ export interface GodotNativeClassInfo {
 export interface GodotCapabilities {
 	native_classes: GodotNativeClassInfo[];
 }
+
+export class AddFuncParams {
+	uri: string;
+	func: string;
+	args?: string[];
+}
```

Comment:
```suggestion
}

```
Missing line termination